### PR TITLE
Outdated example for iOS.

### PR DESCRIPTION
**Pull request by fabioknoedtbcgdv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,6 +299,7 @@ var handle: OpaquePointer?
 
 let status = pv_porcupine_multiple_keywords_init(
     modelFilePath,
+    2, // Number of different keywords to monitor for, or keywordFilePaths.count
```
Comment (kenarsa) [inline README.md]: Let's just use keywordFilePaths.count if possible.

File: README.md [hunk 2]
```diff
@@ -316,13 +317,13 @@ func getNextAudioFrame() -> UnsafeMutablePointer<Int16> {
 
 while true {
     let pcm = getNextAudioFrame()
-    var keyword_index: Int32 = -1
```
Comment (kenarsa) [inline README.md]: Do you think we can just replace "pv_porcupine_process" with "pv_porcupine_multiple_keywords_process"? Then it is coherent with how we initialized the object to do multi-keyword detection above. Thank you.

**Comment by fabioknoedtbcgdv** [comment]
Made the changes you proposed.

**Comment by kenarsa** [comment]
thank you @fabioknoedtbcgdv

**Pull request merged**